Client/Frontend/Browser/URLBarView.swift

```diff
@@ -148,18 +128,35 @@ class URLBarView: UIView {
         return button
     }()
 
-    var menuButton = ToolbarButton()
     var bookmarkButton = ToolbarButton()
     var forwardButton = ToolbarButton()
-    var stopReloadButton = ToolbarButton()
+    var shareButton = ToolbarButton()
+    var addTabButton = ToolbarButton()
+    lazy var menuButton = ToolbarButton().then {
```

Comment:
`lazy`?

Reply:
Because below has a reference to self:

`$0.addTarget(self, action: #selector(didClickMenu), for: .touchUpInside)`